packages/ui/src/styles/components.css

```
@@ -2506,7 +2555,10 @@ a.badge--destructive:hover {
   line-height: 1.25;
   max-width: 18rem;
   outline: none;
+  overflow-wrap: anywhere;
   padding: 0.3125rem 0.5rem;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)